src/js/EditsContainer.jsx

```diff
@@ -2,29 +2,31 @@ var React = require('react');
 var request = require('superagent');
 
 var EditsSyntacticalValidity = require('./EditsSyntacticalValidity.jsx');
+var EditsMacro = require('./EditsMacro.jsx');
 var EditsHeaderDescription = require('./EditsHeaderDescription.jsx');
 
 var EditsContainer = React.createClass({
   getInitialState: function() {
     return {
-    	"syntactical": [],
-      "validity": []
+      edits: {
+        syntactical: [],
+        validity: [],
+        macro: {}
```

Comment:
Cool on the edits ... just want to keep the backend folks in mind too. Might not matter at all for them though.

I agree on the data. We have a good idea about it but not the final. I kind of think we're learning a lot as we move along which could keep leading us toward the final. I feel better about that approach rather than trying to just figure it out. Building the app out has been a great way to keep learning what we really need.